### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by tbergq**
replace @kiwicom/relay with @adeira/relay

### CODE REVIEW

File: scripts/configureApplication.js [hunk 1]
```diff
@@ -66,25 +66,4 @@ if (!fs.existsSync(dotEnvPath)) {
   fs.writeFileSync(dotEnvPath, `SENTRY_DSN=${sentryDsn}`);
 }
 
-log('Patch node crawler');
-
-const files = globSync('/**/jest-haste-map/build/crawlers/node.js', {
```
Comment (tbergq) [inline scripts/configureApplication.js]: I think this should be fixed for all sub-dependencies as well now 🤞

File: packages/localization/package.json [hunk 2]
```diff
@@ -8,11 +8,11 @@
     "@kiwicom/rnmodules": "^0",
     "date-fns": "^1.30.1",
     "form-data": "^2.5.1",
+    "@adeira/fetch": "^1.0.0",
```
Comment (RobinCsl) [inline packages/localization/package.json]: Do you want it as a dep and not as a peerDep like previously?

File: package.json [hunk 3]
```diff
@@ -43,10 +43,10 @@
     "react": "16.8.6",
     "react-native": "0.60.5",
     "react-native-app-registry-components-to-constants": "0.0.1",
+    "@adeira/fetch": "^1.0.0",
```
Comment (RobinCsl) [inline package.json]: Same question here.

File: packages/localization/package.json (hunk 2 above)
Comment (tbergq) [reply packages/localization/package.json]: Yes, it is a dependency, and should be just that instead of peerDependency

**Pull request merged**